packages/frontend/src/hooks/useAddTransaction.ts

```diff
@@ -20,5 +21,7 @@ export const useAddTransaction = () => {
                 serializedTx: serializeTransaction(tx),
             })
         );
+        if (!fathom) return;
+        fathom.trackRegisteredGoal(TX_FATHOM_EVENTS[tx.type], 0);
```

Comment:
Isn't this operation async? Maybe it's worth it to encapsulate this in a try catch block

Reply:
Maybe just a warn log is enough, it's not something that is really stopping the app from working

Reply:
Fixed In c2a0c120e14abab42b64aea69a86403598289f14.